src/components/userProfile/sagas.js

```diff
@@ -0,0 +1,39 @@
+import { effects } from 'redux-saga';
+
+import App from '../app';
+import { fetchUserData } from './actions';
+import { REQUEST_FETCH_USER_DATA } from './actionTypes';
+
+const { selectors: { getUserData } } = App;
+
+function* findUserData(){
```

Comment:
this saga is not needed, use the `getUserData` selector in the component that needs the user data

Reply:
okay